packages/button/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@kaizen/button",
+  "version": "0.0.0",
```

Comment:
Since we're resetting the version, we should probably delete the changelog.
Alt if we want to keep the changelog, we should revert the version to match it.